zerver/tests/test_invite.py

```diff
@@ -146,7 +146,7 @@ def test_query_counts_when_admin_assigns_streams(self) -> None:
 
         new_user = self.create_simple_new_user(realm, new_user_email)
 
-        with self.assert_database_query_count(14):
+        with self.assert_database_query_count(22):
```

Comment:
Yeah because we can't use `select_related` for streams in `PreregistrationUser` to fetch `can_access_stream_topics_group__named_user_group` but I tried `prefetch_related` and it worked. Is it okay to use?

Reply:
Yes, that's because streams is a many to many field and it is fine to use `prefetch_related`.